pinot-common/src/main/java/org/apache/pinot/common/metrics/ServerMeter.java

```diff
@@ -148,6 +148,15 @@ public enum ServerMeter implements AbstractMetrics.Meter {
    * by 2.
    */
   AGGREGATE_TIMES_NUM_GROUPS_LIMIT_REACHED("times", true),
+  /**
+   * Number of times the number of groups has reached the warning limit.
+   * It is increased at most one by one each time per stage.
+   * That means that if a stage has 10 workers and all of them reach the limit, this will be increased by 1.
+   * But if a single query has 2 different aggregate operators and each one reaches the limit, this will be increased
+   * by 2.
+   */
```

Comment:
It looks like this value is only increased in SSE. Could we increase it in MSE as well? We could do the same trick we do with `AggregateOperator.StatKey.NUM_GROUPS_LIMIT_REACHED`: Introduce a new key saying whether we reached the logging limit or not.

We should also log in AggregateOperator in the same way we log in SSE

Reply:
We may also want to log/increase the metric in GroupByCombineOperator and distinct operators?